config/manifests-hub/bases/kernel-module-management-hub.clusterserviceversion.yaml

```diff
@@ -26,13 +26,13 @@ spec:
       deployments: null
     strategy: ""
   installModes:
-  - supported: false
+  - supported: true
```

Comment:
What is the difference between `OwnNamespace` and `SingleNamespace`?

Reply:
[OLM Docs](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#targetnamespaces-and-their-relationship-to-installmodes)